src/main/java/ch/njol/skript/expressions/ExprName.java

```diff
@@ -165,8 +170,8 @@ public String convert(Object o) {
 			if (TITLE_METHOD != null) {
 				try {
 					return (String) TITLE_METHOD.invoke(o);
-				} catch (Throwable e) {
-					Skript.exception(e);
+				} catch (Throwable event) {
+					Skript.exception(event);
```

Comment:
```suggestion
				} catch (Exception e) {
					Skript.exception(e);
```
This isn't an event so it can be `e` for `exception`

Reply:
Probably even better using `ex`

Reply:
> ```suggestion
> 
> 				} catch (Exception e) {
> 
> 					Skript.exception(e);
> 
> ```
> 
> This isn't an event so it can be `e` for `exception`

Oops, how I did not realised..